setup.py

```diff
@@ -90,6 +90,7 @@ def get_git_sha():
         'sqlalchemy',
         'sqlalchemy-utils',
         'sqlparse',
+        'tableschema==1.1.0',
```

Comment:
Don't pin in `setup.py` unless it's strictly required.

Reply:
DOne